### PR TITLE
Track time to download and copy individual chunks

### PR DESCRIPTION
**Pull request by lavjain**

### CODE REVIEW

**Comment by shivzone** [comment]
I agree with @jimmyyih lets record the time conditionally based on the flag level

**Comment by lavjain** [comment]
PR https://github.com/greenplum-db/gpbackup-s3-plugin/pull/20 allows the user to set the log level appropriately.

**Comment by shivzone** [comment]
Its not about the cost of the call. Just that we don't start introducing time tracking in non debug mode for every possible function in the future

**Comment by lavjain** [comment]
@shivzone So your suggestion is to put if condition around `time.Now()`?

**Comment by shivzone** [comment]
Yes. Please make the version check as ` >= LOGVERBOSE` that way log levels `verbose` and `debug` will see them.

File: s3plugin/s3plugin.go [hunk 1]
```diff
@@ -319,6 +332,11 @@ func downloadFile(sess *session.Session, bucket string, fileKey string,
 			finalErr = err
 			break
 		}
+		if gplog.GetLogFileVerbosity() >= gplog.LOGVERBOSE {
+			gplog.Verbose("Downloaded %d bytes (chunk %d) for %s in %v",
+				endByte - startByte + 1, currentChunkNo, filepath.Base(fileKey),
```
Comment (jimmyyih) [inline s3plugin/s3plugin.go:337]: > endByte - startByte + 1

Why is there a `+ 1`?

Comment (lavjain) [reply s3plugin/s3plugin.go]: It was based on an older version of `s3_parallel_download` branch. Gone after rebasing with current master.

**Comment by lavjain** [comment]
Rebased with master

**Pull request merged**